Some identicons not created

As mentioned in #141 there are some identicons which aren't created by pydenticon as the download_image() function doesn't raise an exception once it excepts a URLError.

zimcheck for tezos.stackexchange.com gives the following at the end
```
[ERROR] Invalid external links found :
  I/static/identicon/2271.png (../../I/static/identicon/2271.png) was not found in article A/question/1955.html
  I/static/identicon/4817.png (../../I/static/identicon/4817.png) was not found in article A/question/2180.html
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+### 1.3.1
+
+* fixed identicons for missing source image (#142)
+
 ### 1.3
 
 * better logging

--- a/sotoki/sotoki.py
+++ b/sotoki/sotoki.py
@@ -788,7 +788,7 @@ def markdown(text):
         return "-"
     return text_html
 
-  
+
 def dict_factory(cursor, row):
     d = {}
     for idx, col in enumerate(cursor.description):
@@ -962,6 +962,10 @@ def download_image(url, fullpath, convert_png=False, resize=False):
                 + str(e)
                 + "\n"
             )
+            raise Exception(
+                os.path.basename(fullpath)
+                + " > download_image failed to download the file"
+            )
         else:
             ext = get_filetype(headers, tmp_img)
             try:
@@ -1094,9 +1098,12 @@ def grab_title_description_favicon_lang(url, output_dir, do_old):
     if favicon[:2] == "//":
         favicon = "http:" + favicon
     favicon_out = os.path.join(output_dir, "favicon.png")
-    download_image(
-        favicon, favicon_out, convert_png=True, resize=48,
-    )
+    try:
+        download_image(
+            favicon, favicon_out, convert_png=True, resize=48,
+        )
+    except Exception as e:
+        print(e)
     return [title, description, lang]
 
 

--- a/sotoki/sotoki.py
+++ b/sotoki/sotoki.py
@@ -962,10 +962,7 @@ def download_image(url, fullpath, convert_png=False, resize=False):
                 + str(e)
                 + "\n"
             )
-            raise Exception(
-                os.path.basename(fullpath)
-                + " > download_image failed to download the file"
-            )
+            raise e
         else:
             ext = get_filetype(headers, tmp_img)
             try:

